tests/unit/test_charm.py

```diff
@@ -346,7 +346,7 @@ def test_given_can_connect_to_workload_when_start_simulation_action_then_simulat
         self.harness.charm._on_start_simulation_action(event=event)
 
         patch_exec.assert_any_call(
-            command=["/bin/gnbsim", "--cfg", "/etc/gnbsim/gnb.conf"], timeout=30
+            command=["/bin/gnbsim", "--cfg", "/etc/gnbsim/gnb.conf"], timeout=300
```

Comment:
I'm somewhat unconvinced that this will solve the issue we think it will solve. Whenever we get over 30 seconds, it is typically when the simulation is blocked (something like "sending upstream traffic to UPF"). I haven't seen successful runs that took that long.

Reply:
I am not certain it will solve the issue, but it will let `gnbsim` timeout on its own before we timeout. As it is right now, because we timeout first, we lose all the output and the logs are empty on failure, when they are the most essential.